### PR TITLE
(Fix):Added cluster_id for self connect cluster

### PR DESCRIPTION
**Pull request by amitbhatt818**
- Fetched the cluster_id 
- Created configmap for cluster id

Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

#### Versions:

```
$ Kubernetes version
$ Kubernetes platform
$ kubectl version

```

### CODE REVIEW

File: litmus/director/cluster-connect-check/test.yaml [hunk 1]
```diff
@@ -19,6 +19,16 @@
         - set_fact:
             director_url : "http://{{ director_ip }}:30380"
 
+        - name: Fetch connected cluster id
+          shell: python3 /api_testing/cluster/cluster-connect.py --cluster_name {{ cluster_name }} --provider {{ provider }} --url {{ director_url }} --username {{ username.stdout }} --password {{ password.stdout }}
```
Comment (harshshekhar15) [inline litmus/director/cluster-connect-check/test.yaml]: @amitbhatt818 `cluster_name` and `provider` needs to be defined somewhere.

File: litmus/director/cluster-connect-check/test_vars.yml [hunk 2]
```diff
@@ -2,3 +2,5 @@ test_name: cluster-connect-check
 director_client_namespace: "{{ lookup('env','DIRECTOR_CLIENT_NAMESPACE') }}"
 client_components: ['cortex-agent','dmaas-agent','fluentd-aggregator','fluentd-forwarder','kube-state-metrics','maya-io-agent','openebs-manager','status-agent','upgrade-controller','velero','weave-scope-agent','weave-scope-app','weave-scope-cluster-agent']
 director_ip: "{{ lookup('env','DIRECTOR_IP') }}"
+cluster_name: "{{ lookup('env','CLUSTER_NAME') }}"
+provider: "{{ lookup('env','PROVIDER') }}"
```
Comment (harshshekhar15) [inline litmus/director/cluster-connect-check/test_vars.yml]: @amitbhatt818 `cluster_name` and `provider` are not being passed in env right now, we need to pass it as env to get it here.

File: litmus/director/cluster-connect-check/run_litmus_test.yml [hunk 3]
```diff
@@ -26,7 +26,11 @@ spec:
           - name: ANSIBLE_STDOUT_CALLBACK
             value: default  
           - name: DIRECTOR_CLIENT_NAMESPACE
-            value: maya-system     
+            value: maya-system    
+          - name: PROVIDER
+            value: gcp
+          - name: CLUSTER_NAME
+            value: oep-cluster
```
Comment (harshshekhar15) [inline litmus/director/cluster-connect-check/run_litmus_test.yml]: @amitbhatt818 `CLUSTER_NAME` will be the name with which the self-connect cluster gets connected. `oep-cluster` is the cluster name under which we are connecting an external cluster.

**Comment by amitbhatt818** [comment]
#289

**Pull request closed without merging**